src/System.Windows.Extensions/src/System/Drawing/Printing/MarginsConverter.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Globalization;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace System.Drawing.Printing
+{
+    public class MarginsConverter : ExpandableObjectConverter
+    {
+        private readonly ConstructorInfo _constructorInfo = typeof(Margins).GetTypeInfo().GetConstructor(new Type[] { typeof(int), typeof(int), typeof(int), typeof(int) });
```

Comment:
We do not cache it in other converters, so we should not cache it here either for consistency.

Note that catching it upfront like you have done it will hurt perf in cases where it is not actually used.

Reply:
I'd suggested caching it if perf mattered, but I'm guessing it doesn't (and there's other costly stuff happening anyway). If we do cache it, it should be done lazily. I'm also fine with not caching it at all, both for consistency and simplicity.